manifests/j/JanDeDobbeleer/OhMyPosh/5.6.4/JanDeDobbeleer.OhMyPosh.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: JanDeDobbeleer.OhMyPosh
+PackageVersion: 5.6.4
+PackageLocale: en-US
+Publisher: Jan De Dobbeleer
+PackageName: Oh My Posh
+License: GPL

```

Comment:
Might I suggest adding a bit more metadata?
```suggestion
Publisher: Jan De Dobbeleer
PublisherUrl: https://github.com/JanDeDobbeleer
PublisherSupportUrl: https://github.com/JanDeDobbeleer/oh-my-posh/issues
# PrivacyUrl: 
# Author: 
PackageName: Oh My Posh
PackageUrl: https://ohmyposh.dev/
License: GNU General Public License v3.0
LicenseUrl: https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/COPYING
Copyright: Copyright © 2021 Jan De Dobbeleer and contributors.
CopyrightUrl: https://ohmyposh.dev/
ShortDescription: Prompt theme engine for any shell
Description: Oh My Posh is a custom prompt engine for any shell that has the ability to adjust the prompt string with a function or variable.
# Moniker: 
```

Reply:
Since the package is automated by the CI, you would probably have to submit a PR to https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/packages/winget/JanDeDobbeleer.OhMyPosh.locale.en-US.yaml to make sure that the newer packages have the latest metadata.